app/controllers/application_controller.rb

```diff
@@ -83,4 +83,8 @@ def track_get_requests
 
     analytics.track_event("GET request for #{controller_name}##{action_name}")
   end
+
+  def create_user_event(event_type, user = current_user)
+    Event.create(user_id: user.id, event_type: event_type)
+  end
```

Comment:
I don't think this method should be private. It's called by several controllers and needs to be unit tested since all we're doing in the controller tests is making sure it gets called.